src/api/migraphx.py

```diff
@@ -178,14 +178,67 @@ def shapes(h):
              returns='const migraphx::shape&')
 
 
+@auto_handle()
+def instruction_ref(h):
+    h.constructor('create')
```

Comment:
This constructor can be removed. I think you need to use `pass` to make python happy.

Reply:
Done.